src/clr-core/badge/_properties.scss

```diff
@@ -4,10 +4,15 @@
 
 @include exports('badge.properties') {
   @if $clr-use-custom-properties == true {
-    :root {
+    :root,
+    :host {
       --clr-badge-font-color-light: var(--clr-color-neutral-0);
       --clr-badge-font-color-dark: var(--clr-color-neutral-1000);
 
+      --clr-badge-bg-color: var(--clr-color-neutral-600);
+      --clr-badge-color: var(--clr-badge-font-color-light);
```

Comment:
Looks good! Only request I have is if you don't mind updating the TSDoc @cssprop list in the badge and tag component to show just the properties that they use now

Reply:
Updated and rebased!